migrations/versions/89a031bd9657_remove_friends.py

```diff
@@ -0,0 +1,46 @@
+"""Remove friends
+
+Revision ID: 89a031bd9657
+Revises: 44c4368c2db1
+Create Date: 2023-07-04 18:44:13.041005
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "89a031bd9657"
+down_revision = "44c4368c2db1"
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, "claim_code", ["claim_code"])
+    op.create_foreign_key(None, "print_key", "print_key", ["parent_id"], ["id"])
+    op.create_unique_constraint(None, "printer", ["used_claim_code"])
```

Comment:
not sure why alembic inserted these, but they don't look related so I'm inclined to remove them
```suggestion
```